changelog.d/5686.feature

```diff
@@ -0,0 +1 @@
+Correction of #5674. Use `M_USER_DEACTIVATED` instead of `M_UNKNOWN` for errcode when a deactivated user attempts to login.
```

Comment:
Not sure it's a good idea to include the original PR's number in the newsfile, it would make it more confusing in the release's changelog than bring real value imho, and the rest of it is good enough.

Reply:
Fair enough, especially since these PRs are now separated by a release.